tests/client/test_client_task_meilisearch.py

```diff
@@ -167,3 +167,5 @@ def test_delete_all_tasks(client):
         "statuses=succeeded%2Cfailed%2Ccanceled"
         in tasks_after["results"][0]["details"]["originalFilter"]
     )
+    assert len(tasks_after["results"]) >= 1
+    assert len(tasks_before["results"]) == len(tasks_after["results"])
```

Comment:
These are repeated asserts from lines 164-165 so the problem will still be here. The root of the problem is `assert len(tasks_before["results"]) == len(tasks_after["results"])` is not guaranteed to be true.

So you can remove lines 155, 165, 170, and 171. If `assert "statuses=succeeded%2Cfailed%2Ccanceled" in tasks_after["results"][0]["details"]["originalFilter"]` we know the delete was successful.

Reply:
I have done the suggested changes, can you look into it again?